src/main/java/com/github/pjfanning/xlsx/impl/ooxml/OoXmlStrictConverter.java

```diff
@@ -0,0 +1,227 @@
+package com.github.pjfanning.xlsx.impl.ooxml;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventFactory;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLEventWriter;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.Attribute;
+import javax.xml.stream.events.EndElement;
+import javax.xml.stream.events.Namespace;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import org.apache.poi.hssf.usermodel.HSSFAnchor;
+import org.apache.poi.hssf.usermodel.HSSFDataBarFormatting;
+import org.apache.poi.hssf.usermodel.HSSFDateUtil;
+
+public class OoXmlStrictConverter implements AutoCloseable {
+
+    private static final XMLEventFactory XEF = XMLEventFactory.newInstance();
```

Comment:
there is a class called org.apache.poi.util.XMLHelper has methods to create the XML input and output factories - can you use those?

Reply:
Done